src/Tasks/Common/Resources/Strings.resx

```diff
@@ -506,8 +506,11 @@ The following are names of parameters or literal values and should not be transl
     <value>NETSDK1100: Windows is required to build Windows desktop applications.</value>
     <comment>{StrBegin="NETSDK1100: "}</comment>
   </data>
-  <data name="ILLink_Info" xml:space="preserve">
-    <value>Optimizing assemblies for size, which may change the behavior of the app. Be sure to test after publishing. See: https://aka.ms/dotnet-illink</value>
+  <data name="ILLinkTimeAwareness" xml:space="preserve">
+    <value>Running ILLink, this process might take a while</value>
```

Comment:
I'd avoid mentioning ILLink here - maybe "Optimizing assemblies for size. This might take a while."?